.github/workflows/ci.yaml

```diff
@@ -45,5 +59,7 @@ jobs:
         run: |
           ./scripts/set-version.sh "${{ matrix.spark_version }}" "${{ matrix.scala_version }}"
           git diff
+      - name: Fetch mvn dependencies
+        run: mvn --batch-mode dependency:go-offline
       - name: Build
-        run: mvn install
+        run: mvn --batch-mode install
```

Comment:
How hard would it be to add a call to?:
```
scripts/createImage.sh -s "sparkhome"
```
that script fails in master right now with spark 3.5, because of some pom changes in our master.

that failure would have been caught earlier if we could have tested it here.

Reply:
I have added that.

Reply:
Scala 2.13 now successfully builds the image